lib/Basics/operating-system.h

```diff
@@ -705,9 +702,10 @@
 
 //#define _CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES     1
 
-#include <WinSock2.h>
+#include <WinSock2.h>  // must be before windows.h
 #include <io.h>
 #include <stdio.h>
+#include <windows.h>
```

Comment:
do we really need windows.h? at least in this place I strongly dislike it.

Reply:
It seems you are correct and it is unneeded here.